server/homebrew.api.js

```diff
@@ -4,13 +4,36 @@ const router = require('express').Router();
 const zlib = require('zlib');
 const GoogleActions = require('./googleActions.js');
 const Markdown = require('../shared/naturalcrit/markdown.js');
+const yaml = require('js-yaml');
 
 // const getTopBrews = (cb) => {
 // 	HomebrewModel.find().sort({ views: -1 }).limit(5).exec(function(err, brews) {
 // 		cb(brews);
 // 	});
 // };
 
+const mergeBrewText = (brew)=>{
+	let text = brew.text;
+	if(brew.style !== undefined) {
+		text = `\`\`\`css\n` +
+			`${brew.style || ''}\n` +
+			`\`\`\`\n\n` +
+			`${text}`;
+	}
+	const metadata = {
+		title       : brew.title,
+		description : brew.description,
+		tags        : brew.tags,
+		systems     : brew.systems,
+		renderer    : brew.renderer
+	};
```

Comment:
This works fine. I wonder if it looks any cleaner to use https://lodash.com/docs/#pick ?

Reply:
Since the values in the object share the name between instances yes it'd definitely be cleaner to use pick -- I'll update that.